crates/uv-python/src/installation.rs

```diff
@@ -180,6 +191,21 @@ impl PythonInstallation {
         installed.ensure_externally_managed()?;
         installed.ensure_sysconfig_patched()?;
         installed.ensure_canonical_executables()?;
+
+        let minor_version = installed.minor_version_key();
+        let highest_patch = installations
+            .find_all()?
+            .filter(|installation| installation.minor_version_key() == minor_version)
+            .filter_map(|installation| installation.version().patch())
+            .fold(0, std::cmp::max);
+        if installed
+            .version()
+            .patch()
+            .is_some_and(|p| p >= highest_patch)
+        {
+            installed.ensure_minor_version_link(preview)?;
```

Comment:
uv checks for the highest patch for each minor version key and ensures that the corresponding minor version link exists and points to it (but a symlink directory is only initially created in preview mode).